escalation-eda/escalation-swf/README.md

```diff
@@ -149,6 +169,7 @@ echo $SWF_INSTANCE_ID
 Where [input.json](./input.json) defines the input document as:
 ```json
 {
+  "notifications_secret": "${NOTIFICATIONS_SECRET}",
   "namespace": "${NAMESPACE}",
   "email": {
     "manager": "${MANAGER}"
```

Comment:
you can probably remove the whole `notification` part from the input.json now, IIUC

Reply:
Thanks. Sure, done.